index.bs

```diff
@@ -0,0 +1,565 @@
+<pre class="metadata">
```

Comment:
Consider adding a [`.pr-preview.json`](https://github.com/tobie/pr-preview#configuration-file) in the same change.